bankayma_account/views/templates.xml

```diff
@@ -282,7 +282,7 @@
                 <a
                     class="btn btn-lg btn-primary"
                     aria-label="New Vendor Bill"
-                    href="/my/invoices/new"
+                    href="redirect=%2Fmy%2Faccount%3Fredirect%3D%2Fmy%2Finvoices%2Fnew"
```

Comment:
@toyrobot are you sure you *always* want this link to go via the account details? If so, the link should be `/my/account?redirect=/my/invoices/new` - but I guess this rather should check if fpos or some other value(s) are set, and go directly to the new invoices in that case?

Reply:
ah, i reacted on an outdated diff, so the link is correct, but still the question about always wanting this remains